Makefile

```diff
@@ -5,12 +5,12 @@ IMAGE_REPO ?= nmstate
 NAMESPACE ?= nmstate
 
 HANDLER_IMAGE_NAME ?= kubernetes-nmstate-handler
-HANDLER_IMAGE_SUFFIX ?=
+HANDLER_IMAGE_SUFFIX ?= :latest
```

Comment:
I'd like to see this renamed to HANDLER_IMAGE_TAG then. And probably not have the ':' as part of it.

Reply:
In case you want to push specific hashes instead of latest you can do `HANDLER_IMAGE_SUFFIX=@sha256:[id] `